web/pdf_presentation_mode.js

```diff
@@ -119,17 +119,31 @@ var PDFPresentationMode = (function PDFPresentationModeClosure() {
       return true;
     },
 
-    /**
-     * Switches page when the user scrolls (using a scroll wheel or a touchpad)
-     * with large enough motion, to prevent accidental page switches.
-     * @param {number} delta - The delta value from the mouse event.
-     */
-    mouseScroll: function PDFPresentationMode_mouseScroll(delta) {
+    _mouseWheel: function PDFPresentationMode_mouseWheel(evt) {
       if (!this.active) {
         return;
       }
```

Comment:
I think that you need `evt.preventDefault;` here, since otherwise it's possible to zoom the viewer UI while Presentation Mode is active (if e.g. `Ctrl` is held down while using the mouse wheel).